### PR TITLE
chore(deps): Update to latest airswap dependencies

### PR DESCRIPTION
**Pull request by Mitsichury**

### CODE REVIEW

File: src/model/DbOrder.ts [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-}
\ No newline at end of file
+export type DbOrder = Modify<FullOrderERC20, {
+    expiry: number;
+    approximatedSignerAmount: BigInt;
+    approximatedSenderAmount: BigInt;
+}>
+
+export type DbOrderParty = Modify<OrderParty, {
+    amount: number;
```
Comment (piersss) [inline src/model/DbOrder.ts:18]: This also has to be a `BigInt` then.

**Review by piersss — CHANGES_REQUESTED** [review]
Some small feedback

File: src/client/getHealthCheck.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    registry: string, 
+    network: number, 
+    peers: string[], 
+    databaseOrders: number
+}
+
+export async function getHealthCheck(host: string): Promise<HealthCheckResponse> {
+    try {
+      const response = (await axios.get(host)) as AxiosResponse<JsonRpcResponse<any>>
+      return Promise.resolve(response.data.result as unknown as any)
```
[the remark this answers is not in the document]
Comment (Mitsichury) [reply src/client/getHealthCheck.ts]: Done thank you :)

File: src/model/DbOrder.ts (hunk 1 above)
Comment (Mitsichury) [reply src/model/DbOrder.ts]: Done thank you :)

File: src/client/Web3SwapERC20Client.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        contract.on("Cancel", (nonce, signerWallet) => {
+            this.onEvent(nonce, signerWallet);
+        });
+        this.contracts.push(contract);
+        this.registeredChains.push(String(chainId));
+        console.log("Registered event from chain", chainId)
+    }
+
+    private keyExists(network: string): boolean {
+        return this.registeredChains.indexOf(network) !== -1;
```
[the remark this answers is not in the document]
Comment (Mitsichury) [reply src/client/Web3SwapERC20Client.ts]: Done thank you :)

**Review by piersss — APPROVED** [review]
LGTM

**Pull request merged**